docs/essentials/actions.md

```diff
@@ -12,7 +12,7 @@ Actions works via supplying special Storybook-generated “action” arguments (
 
 #### Action argType annotation
 
-You can use [argTypes](../api/stories#argtypes) to tell Storybook that an arg to your story should be an action. Usually it makes sense to do this at the component level (although it can be done per story):
+You can use [argTypes](../api/mdx.md#argtypes) to tell Storybook that an arg to your story should be an action. Usually it makes sense to do this at the component level (although it can be done per story):
```

Comment:
This should link to https://docs.google.com/document/d/1BawdfqGhsP5tqhYQbVn29c6_rAz1ePmk9QayoXLDOKM/edit#

I'm not sure where that went? Looks like it was accidentally dropped in https://github.com/storybookjs/storybook/pull/11751

Reply:
fixed with the latest commits. Let me know @tmeasday if you're ok with it